h2/src/test/org/h2/test/scripts/TestScript.java

```diff
@@ -204,7 +204,7 @@ public void test() throws Exception {
                 "parsedatetime", "quarter", "second", "truncate", "week", "year", "date_trunc" }) {
             testScript("functions/timeanddate/" + s + ".sql");
         }
-        for (String s : new String[] { "lead", "nth_value", "ntile", "row_number" }) {
+        for (String s : new String[] { "lead", "nth_value", "ntile", "ratio_to_report", "row_number" }) {
```

Comment:
forgot to 'git add' file?

Reply:
Yes, I did. Thanks for notice, missing SQL test cases do not cause quick termination of TestAll.